app/parse.py

```diff
@@ -15,9 +39,61 @@ class Product:
     num_of_reviews: int
 
 
+def parse_single_product(product: WebElement):
+    return Product(
+        title=product.find_element(By.CLASS_NAME, "title").text,
+        description=product.find_element(By.CLASS_NAME, "description").text,
+        price=float(product.find_element(By.CLASS_NAME, "price").text.replace("$", "")),
+        rating=len(product.find_elements(By.CLASS_NAME, "ws-icon-star")),
+        num_of_reviews=int(
+            product.find_element(By.CLASS_NAME, "review-count").text.split()[0]
```

Comment:
Consider adding error handling for the conversion of review count to int. If the text is not in the expected format, this could raise a ValueError.